src/scripts/main.js

```diff
@@ -1 +1,86 @@
 'use strict';
+
+const result = document.querySelector('body');
+
+function createPromise() {
+  const resolver = (resolve, reject) => {
+    result.addEventListener('click', () => {
+      resolve();
+    });
+
+    setTimeout(() => {
+      reject();
+    }, 3000);
+  };
+
+  return new Promise(resolver);
+}
+
+function createSecond() {
+  const resolver = (resolve, reject) => {
+    result.addEventListener('click', () => {
+      resolve();
+    });
+
+    result.addEventListener('contextmenu', (e) => {
+      e.preventDefault();
+      resolve();
+    });
+  };
+
+  return new Promise(resolver);
+}
+
+function createThird() {
+  const resolver = (resolve) => {
+    let lefttButtonClicked = false;
+    let rightButtonClicked = false;
+
+    result.addEventListener('click', () => {
+      lefttButtonClicked = true;
```

Comment:
```suggestion
      leftButtonClicked = true;
```
Typo with doulbe "t"